lib/instructor/openai/client.rb

```diff
@@ -0,0 +1,44 @@
+OpenAI.configure do |config|
+  config.access_token = ENV.fetch('OPENAI_API_KEY')
+end
+
+module Instructor
+  module OpenAI
+    class Client
+      def initialize
+        @client = ::OpenAI::Client.new do |f|
+          f.response :logger, Logger.new($stdout), bodies: true if ENV['OPENAI_LOG'] == 'debug'
+        end
+      end
+
+      def chat(parameters:, response_model:)
```

Comment:
its not `chat.completions` ?

Reply:
The OpenAI ruby client lib just uses `chat`. For example:

```Ruby
client = OpenAI::Client.new
response = client.chat(
    parameters: {
        model: "gpt-3.5-turbo", 
        messages: [{ role: "user", content: "Hello!"}], 
    })
puts response.dig("choices", 0, "message", "content")
# => "Hello! How may I assist you today?"
    })
```
So I just followed the same pattern. I know the python OpenAI does it differently.

Reply:
Icy. Perfect. That's all I wanted to know.